src/components/Login.js

```diff
@@ -0,0 +1,87 @@
+import React, { Fragment, useState } from 'react';
+import { Form, Button, Row, Col } from 'react-bootstrap';
+
+const Login = () => {
+  const [user, setUser] = useState({
+    email: '',
+    password: ''
+  });
+
+  const { email, password } = user;
+
+  const onChange = e => {
+    setUser({ ...user, [e.target.name]: e.target.value });
+  };
+
+  const onSubmit = e => {
+    e.preventDefault();
+    if (email === '' || password === '') alert('Please fill all fields!');
```

Comment:
Zamień może tego alerta na tekst, tak jak w wymaganiach.

Reply:
Ok, chociaż to i tak jest tylko tymczasowe